pyproject.toml

```
@@ -10,7 +10,7 @@ authors = [
   { name = "trag1c", email = "dev@jakubr.me" },
 ]
 dependencies = [
-  "discord-py~=2.5.0",
+  "discord-py~=2.6.0",
```

This should probably be:

```suggestion
  "discord-py>=2.6.2,<2.7",
```

Due to https://github.com/Rapptz/discord.py/issues/10280.